test/SlowTests/Issues/RavenDB-17597.cs

```diff
@@ -0,0 +1,360 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Reflection;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using FastTests;
+using FastTests.Client;
+using NuGet.ContentModel;
+using Orders;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Indexes;
+using Raven.Client.Documents.Indexes.Spatial;
+using Raven.Client.Documents.Linq;
+using Raven.Client.Documents.Operations;
+using Raven.Client.Documents.Operations.Indexes;
+using Raven.Client.Documents.Session;
+using Raven.Client.Exceptions.Database;
+using Raven.Client.Exceptions.Documents.Indexes;
+using Raven.Client.ServerWide;
+using Raven.Client.ServerWide.Operations;
+using Raven.Server;
+using Raven.Server.Commercial;
+using Raven.Server.Config;
+using Raven.Server.Documents;
+using Raven.Server.Documents.Indexes;
+using Raven.Server.ServerWide.Context;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+using Raven.Client;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_17597 : ClusterTestBase
+    {
+        public RavenDB_17597(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public async Task CopyToIndexDefinitionWorksProperly()
```

Comment:
please always remember to check if you do not have warnings, it is as easy as going to https://github.com/ravendb/ravendb/pull/14611/files after GH actions completed

Reply:
please fix the warning here